src/terminal/Terminal.cpp

```diff
@@ -1677,4 +1681,23 @@ void Terminal::setActiveStatusDisplay(ActiveStatusDisplay activeDisplay)
     }
 }
 
+bool Terminal::isHighlighted(CellLocation const& _cell) const noexcept
+{
+    return isHighlightActive && highlightRange_.has_value() && isInRange(highlightRange_.value(), _cell);
+}
+void Terminal::updateHighlightVisibilityState()
```

Comment:
Empty like above, for keeping style a bit :)

Reply:
you mean an empty line at above `updateHighlightVisibilityState`?

Reply:
Done